pages/gallery/[galleryId]/collection/create.tsx

```diff
@@ -34,9 +39,15 @@ function LazyLoadedCollectionEditor({ galleryId }: Props) {
   );
 
   const track = useTrack();
+  const reportError = useReportError();
+  const [, setGeneralError] = useState('');
```

Comment:
what's the purpose of tracking this? seems like we don't actually use the value anywhere

ah, are you thinking we can show it in the UI somewhere?

Reply:
I'm planning to show this error somewhere since we show it before. 

Previously it was in the collection name/description modal. 

I'll put it inside the caption box whenever user click the save button. Open to any idea for this placement. 

<img width="713" alt="CleanShot 2022-11-07 at 14 02 16@2x" src="https://user-images.githubusercontent.com/4480258/200237035-4e9ef6b7-72e5-4079-a0b8-8e901fcdfb18.png">